Benchmarks for quantized LSTM (static)

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32335 Benchmarks for quantized LSTM (static)**
* #32074 Static quantization for LSTM (front)
* #31934 Static quantization for LSTM (back)

Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Benchmarks for quantized LSTM (static)

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit b247ac6d:


**None of the build failures appear to be your fault**.

* **1/1** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=b247ac6d8d41b78f1eb692bebb1f1f838f854b56).

### :snowflake: 1 failure recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4287786) pytorch_linux_xenial_py2_7_9_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=87572302) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=87572302)) :snowflake:
<details>
<summary>
<code>Jan 17 02:36:55 AssertionError: tensor(0.3070, grad_fn=<MaxBackward1>) not less than or equal to 0.2 : H not equal: tensor([[[ 1.6992e-01,  2.3584e-01,  3.3769e-03,  ...,  1.6378e-01,</code>
</summary>

```
Jan 17 02:36:55 ====================================================================== 
Jan 17 02:36:55 FAIL [0.179s]: test_lstm (__main__.TestStaticRNN) 
Jan 17 02:36:55 ---------------------------------------------------------------------- 
Jan 17 02:36:55 Traceback (most recent call last): 
Jan 17 02:36:55   File "test_quantized.py", line 1128, in test_lstm 
Jan 17 02:36:55     message="H not equal: {} vs. {}".format(f_h, q_h)) 
Jan 17 02:36:55   File "/var/lib/jenkins/workspace/test/common_utils.py", line 859, in assertEqual 
Jan 17 02:36:55     assertTensorsEqual(x, y) 
Jan 17 02:36:55   File "/var/lib/jenkins/workspace/test/common_utils.py", line 829, in assertTensorsEqual 
Jan 17 02:36:55     self.assertLessEqual(max_err, prec, message) 
Jan 17 02:36:55 AssertionError: tensor(0.3070, grad_fn=<MaxBackward1>) not less than or equal to 0.2 : H not equal: tensor([[[ 1.6992e-01,  2.3584e-01,  3.3769e-03,  ...,  1.6378e-01, 
Jan 17 02:36:55           -4.4926e-02, -3.7548e-02], 
Jan 17 02:36:55          [ 2.1429e-02,  1.9415e-01,  2.0122e-02,  ..., -6.2661e-02, 
Jan 17 02:36:55           -4.8124e-02,  4.6948e-02], 
Jan 17 02:36:55          [-7.7821e-02,  1.3783e-01,  2.9426e-01,  ..., -9.2423e-02, 
Jan 17 02:36:55           -6.3942e-02,  1.4905e-02], 
Jan 17 02:36:55          ..., 
Jan 17 02:36:55          [-5.8719e-02,  7.4594e-02,  2.3528e-01,  ...,  4.2940e-01, 
Jan 17 02:36:55           -2.9867e-02,  3.9597e-02], 
Jan 17 02:36:55          [ 9.3161e-04,  1.4929e-02, -1.2242e-02,  ...,  1.5895e-01, 
Jan 17 02:36:55            1.7733e-01,  1.0385e-01], 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

mingzhe09088 reviewed (approved):
LGTM!

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)

Commit:
Update on "Benchmarks for quantized LSTM (static)"


Differential Revision: [D19443067](https://our.internmc.facebook.com/intern/diff/D19443067)
